client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/index.js

```diff
@@ -72,12 +78,27 @@ const getNormalizationStatus = ( {
 	if ( hasNonEmptyLeaves( errors ) || ( isNormalized && ! normalized ) || ! isNormalized ) {
 		return { isError: true };
 	}
+	if ( ! checkAddressCharacters( values ) ) {
+		return { isWarning: true };
+	}
 	if ( isNormalized ) {
 		return isEqual( values, normalized ) ? { isSuccess: true } : { isWarning: true };
 	}
 	return {};
 };
 
+const hasNonAsciiCharacters = str => /[^\u0000-\u007f]/.test(str);
+
+const checkAddressCharacters = function ( addressValues ) {
```

Comment:
It might be good to change this to an arrow function just to stay consistent with the other functions in this file.